### PR TITLE
Add err.sh for invalid getStaticPaths return value

### PR DESCRIPTION
**Pull request by ijjk**
This adds an err.sh link for an invalid return value from `getStaticPaths` to better explain the error and how to correct it

### CODE REVIEW

**Comment by ijjk** [comment]
# Stats from current PR

<details>
<summary><strong>Default Server Mode</strong></summary>

<br/>

<details>
<summary><strong>General</strong> Overall increase ⚠️</summary>

|  | zeit/next.js canary  | ijjk/next.js add/staticpaths-errsh | Change |
| - | - | - | - |
| buildDuration | 9.6s | 9.8s | ⚠️ +183ms |
| nodeModulesSize | 54.5 MB | 54.5 MB | ⚠️ +188 B |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | zeit/next.js canary  | ijjk/next.js add/staticpaths-errsh | Change |
| - | - | - | - |
| main-HASH.js gzip | 5.05 kB | 5.05 kB | ✓ |
| webpack-HASH.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..9ee3.js gzip | 12.2 kB | 12.2 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 57.2 kB | 57.2 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | zeit/next.js canary  | ijjk/next.js add/staticpaths-errsh | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 4.14 kB | 4.14 kB | ✓ |
| webpack-HASH..dule.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..dule.js gzip | 9.2 kB | 9.2 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 53.2 kB | 53.2 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | zeit/next.js canary  | ijjk/next.js add/staticpaths-errsh | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 20.3 kB | 20.3 kB | ✓ |
| Overall change | 20.3 kB | 20.3 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | zeit/next.js canary  | ijjk/next.js add/staticpaths-errsh | Change |
| - | - | - | - |
| _app.js gzip | 1.06 kB | 1.06 kB | ✓ |
| _error.js gzip | 2.98 kB | 2.98 kB | ✓ |
| hooks.js gzip | 664 B | 664 B | ✓ |
| index.js gzip | 222 B | 222 B | ✓ |
| link.js gzip | 1.9 kB | 1.9 kB | ✓ |
| routerDirect.js gzip | 279 B | 279 B | ✓ |
| withRouter.js gzip | 278 B | 278 B | ✓ |
| Overall change | 7.38 kB | 7.38 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages Modern</strong></summary>

|  | zeit/next.js canary  | ijjk/next.js add/staticpaths-errsh | Change |
| - | - | - | - |
| _app.module.js gzip | 577 B | 577 B | ✓ |
| _error.module.js gzip | 2.08 kB | 2.08 kB | ✓ |
| hooks.module.js gzip | 370 B | 370 B | ✓ |
| index.module.js gzip | 212 B | 212 B | ✓ |
| link.module.js gzip | 1.48 kB | 1.48 kB | ✓ |
| routerDirect..dule.js gzip | 271 B | 271 B | ✓ |
| withRouter.m..dule.js gzip | 270 B | 270 B | ✓ |
| Overall change | 5.26 kB | 5.26 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | zeit/next.js canary  | ijjk/next.js add/staticpaths-errsh | Change |
| - | - | - | - |
| _buildManifest.js gzip | 61 B | 61 B | ✓ |
| _buildManife..dule.js gzip | 61 B | 61 B | ✓ |
| Overall change | 122 B | 122 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | zeit/next.js canary  | ijjk/next.js add/staticpaths-errsh | Change |
| - | - | - | - |
| index.html gzip | 925 B | 925 B | ✓ |
| link.html gzip | 934 B | 934 B | ✓ |
| withRouter.html gzip | 923 B | 923 B | ✓ |
| Overall change | 2.78 kB | 2.78 kB | ✓ |

</details>

</details>
<hr/>
<details>
<summary><strong>Serverless Mode</strong> (Increase detected ⚠️)</summary>

<br/>

<details>
<summary><strong>General</strong> Overall increase ⚠️</summary>

|  | zeit/next.js canary  | ijjk/next.js add/staticpaths-errsh | Change |
| - | - | - | - |
| buildDuration | 10.3s | 10.4s | ⚠️ +56ms |
| nodeModulesSize | 54.5 MB | 54.5 MB | ⚠️ +188 B |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | zeit/next.js canary  | ijjk/next.js add/staticpaths-errsh | Change |
| - | - | - | - |
| main-HASH.js gzip | 5.05 kB | 5.05 kB | ✓ |
| webpack-HASH.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..9ee3.js gzip | 12.2 kB | 12.2 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 57.2 kB | 57.2 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | zeit/next.js canary  | ijjk/next.js add/staticpaths-errsh | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 4.14 kB | 4.14 kB | ✓ |
| webpack-HASH..dule.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..dule.js gzip | 9.2 kB | 9.2 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 53.2 kB | 53.2 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | zeit/next.js canary  | ijjk/next.js add/staticpaths-errsh | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 20.3 kB | 20.3 kB | ✓ |
| Overall change | 20.3 kB | 20.3 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | zeit/next.js canary  | ijjk/next.js add/staticpaths-errsh | Change |
| - | - | - | - |
| _app.js gzip | 1.06 kB | 1.06 kB | ✓ |
| _error.js gzip | 2.98 kB | 2.98 kB | ✓ |
| hooks.js gzip | 664 B | 664 B | ✓ |
| index.js gzip | 222 B | 222 B | ✓ |
| link.js gzip | 1.9 kB | 1.9 kB | ✓ |
| routerDirect.js gzip | 279 B | 279 B | ✓ |
| withRouter.js gzip | 278 B | 278 B | ✓ |
| Overall change | 7.38 kB | 7.38 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages Modern</strong></summary>

|  | zeit/next.js canary  | ijjk/next.js add/staticpaths-errsh | Change |
| - | - | - | - |
| _app.module.js gzip | 577 B | 577 B | ✓ |
| _error.module.js gzip | 2.08 kB | 2.08 kB | ✓ |
| hooks.module.js gzip | 370 B | 370 B | ✓ |
| index.module.js gzip | 212 B | 212 B | ✓ |
| link.module.js gzip | 1.48 kB | 1.48 kB | ✓ |
| routerDirect..dule.js gzip | 271 B | 271 B | ✓ |
| withRouter.m..dule.js gzip | 270 B | 270 B | ✓ |
| Overall change | 5.26 kB | 5.26 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | zeit/next.js canary  | ijjk/next.js add/staticpaths-errsh | Change |
| - | - | - | - |
| _buildManifest.js gzip | 61 B | 61 B | ✓ |
| _buildManife..dule.js gzip | 61 B | 61 B | ✓ |
| Overall change | 122 B | 122 B | ✓ |

</details>

<details>
<summary><strong>Serverless bundles</strong> Overall increase ⚠️</summary>

|  | zeit/next.js canary  | ijjk/next.js add/staticpaths-errsh | Change |
| - | - | - | - |
| _error.js gzip | 290 kB | 290 kB | -255 B |
| 404.html gzip | 1.33 kB | 1.33 kB | ✓ |
| hooks.html gzip | 964 B | 964 B | ✓ |
| index.js gzip | 290 kB | 290 kB | ⚠️ +403 B |
| link.js gzip | 297 kB | 297 kB | ⚠️ +176 B |
| routerDirect.js gzip | 296 kB | 298 kB | ⚠️ +1.64 kB |
| withRouter.js gzip | 298 kB | 298 kB | -30 B |
| Overall change | 1.47 MB | 1.47 MB | ⚠️ +1.94 kB |

</details>

</details>


Commit: a6a672471bfdd5737a9f48d489e37e570ab3f4ab

**Review by timneutkens — APPROVED** [review]

File: errors/invalid-getstaticpaths-value.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+export async function unstable_getStaticProps() {
+  return {
+    paths: Array<string | { params: { [key: string]: string } }>
+  }
+}
+```
+
+### Useful Links
+
+- [AMP HTML Specification](https://www.ampproject.org/docs/fundamentals/spec)
```
Comment (timneutkens) [inline errors/invalid-getstaticpaths-value.md]:
```suggestion
```

File: errors/invalid-getstaticpaths-value.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+- [Possible future syntax](https://github.com/ampproject/amphtml/issues/21600)
```
Comment (timneutkens) [inline errors/invalid-getstaticpaths-value.md]:
```suggestion
```

File: errors/invalid-getstaticpaths-value.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+```js
+export async function unstable_getStaticProps() {
+  return {
+    paths: Array<string | { params: { [key: string]: string } }>
+  }
+}
+```
+
+### Useful Links
```
Comment (timneutkens) [inline errors/invalid-getstaticpaths-value.md]:
```suggestion
```

**Pull request merged**